ng-pkgs/activation_service/package.py

```diff
@@ -0,0 +1,48 @@
+import os
+import sys
+
+import gevent
+
+from jumpscale.loader import j
+
+current_full_path = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(current_full_path + "/sals/")
+from activation_sal import create_gevent_pool
+
+
+class activation_service:
+    def install(self, **kwargs):
+        if "activation_wallet" not in j.clients.stellar.list_all():
+            secret = kwargs.get("secret", None)
+            network = kwargs.get("network", "TEST")
+            wallet = j.clients.stellar.new("activation_wallet", secret=secret, network=network)
```

Comment:
Can't we use an install argument  to specify the walletname to use instead of having a hardcoded one?

Reply:
the package installation in jsng doesn't take arguments. How about we get the wallet name from an env variable that is set manually. That way the wallet name could be configurable?

Reply:
wallet now can be passed with `wallet`